config.json

```diff
@@ -1282,6 +1282,20 @@
         "practices": [],
         "prerequisites": [],
         "difficulty": 3
+      },
+      {
+        "slug": "bottle-song",
+        "name": "Bottle Song",
+        "uuid": "0c786feb-8653-4215-9281-2e1c83afeefe",
+        "practices": [],
+        "prerequisites": [],
+        "difficulty": 3,
+        "topics": [
```

Comment:
Topics are no longer used by Exercism, so you can omit the key for new exercises

Reply:
fixed